app/renderer/components/main/siteInfo.js

```diff
@@ -93,10 +90,9 @@ class SiteInfo extends React.Component {
       </div>
     } else {
       // insecure
-      return <div className={css(styles.secureIcon__wrapper)}>
+      return <div className={css(styles.secureIcon)}>
         <span className={cx({
-          fa: true,
-          'fa-unlock': true,
+          [globalStyles.appIcons.unlock]: true,
```

Comment:
Wondering if we would create components like `<AppIcon unlock custom={styles.secureIcon__fa} />`

Reply:
I would prefer having the icons instead of a FA class. Food for thought if you want to go ahead with this. we could gradually remove them in favor of SVG. Sources can be found here: https://github.com/encharm/Font-Awesome-SVG-PNG